src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -133,25 +143,27 @@ static bool Compare(List<TElem> a, List<TElem> b, ValueComparer<TElem> elementCo
 
             static List<TElem> Snapshot(List<TElem> source, ValueComparer<TElem> elementComparer)
             {
-                if (source == null)
+                if (source is null)
```

Comment:
Is there any specific reason to convert `x == null` to `x is null`? It makes sense if you use pattern matching (i.e. `x is null y`) but just for null checking? I'd stick to the current standard syntax rather than starting to change the entire codebase etc.

Reply:
That's a fair point. I tend to make these changes out of habit more than anything else. 

I'll revert this in an upcoming commit. (This type mapper still lacks type-casting for the array literals.)